modules/test/playwright/fixtures/featureFlagsTest.ts

```diff
@@ -173,20 +169,30 @@ async function invokeServer<T>(
 					}),
 					method: 'POST',
 				})
-					.then((response) => response.text())
-					.then((text) => {
-						const json = JSON.parse(text);
-
-						if (json.error) {
-							resolve({
-								error: json.error,
-							});
-						}
-						else {
-							resolve({
-								result: json,
-							});
+					.then(async (response) => {
+						if (!response?.ok) {
```

Comment:
Typo on 'occured' (should be 'occurred'). 

Also I don't think we need to log the http code, so we could just do:

```javascript
if (!response?.ok) {
	if (response?.status === 404) {
		reject(
			`Feature Flag "${partialBody['key']}" is not available`
		);
	}
	else if (response?.status === 500) {
		reject(
			"An unexpected error has occurred"
		);
	}
	else {
		reject(
			`Unable to fetch feature flag "${partialBody['key']}"`
		);
	}
}
```

thoughts @ethib137?

Reply:
Yep, this looks good to me.

Reply:
done